Assets/Scripts/Ingredients/InteractableBehaviour.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class InteractableBehaviour : MonoBehaviour
+{
+    [SerializeField] GameObject hoverObject;
+    Transform worldCanvas => GameObject.Find("WorldCanvas").transform;
+    [SerializeField] Transform player;
+    GameObject hoverObjectInstance;
```

Comment:
```suggestion
    [SerializeField] private GameObject hoverObject;
    private Transform worldCanvas => GameObject.Find("WorldCanvas").transform;
    [SerializeField] private Transform player;
    private GameObject hoverObjectInstance;
```

Reply:
Is denk wel handiger om neer te zetten wanneer een variable public of private is, zodat als een ander het nodig heeft in een ander script, dan ook gelijk kan aflezen wat die kan gebruiken. 

Wannee je niks neerzet is het default private, maar is denk netter om het er gewoon bij te zetten of het public of private is.